sysroot/usr/include/kernel/apps/primes.hpp

```diff
@@ -1,4 +1,27 @@
+/**
+ * @file primes.hpp
+ * @author Micah Switzer (mswitzer@cedarville.edu)
+ * @brief Prime computation tasks
+ * @version 0.1
+ * @date 2021-06-05
```

Comment:
I know you just added these headers, but would it be worth it to put the original author date of these files?

Reply:
Sure, I can do that.

Reply:
Resolved in a5c787b.